src/components/Layouts/components/Notification/Notification.tsx

```diff
@@ -30,18 +34,36 @@ const Notification = () => {
     }
   }, [])
 
-  const handleMarkAllAsRead = useCallback(() => {
+  const handleMarkAllAsRead = useCallback(async () => {
     if (notifications) {
       const updatedNotifications = notifications.map((notification) => ({
         ...notification,
         seen: true
       }))
       setNotifications(updatedNotifications)
+      await markAllNotificationAPI()
```

Comment:
using await in the end of function like this is redundant. Same as the async function